src/Compilers/Core/CodeAnalysisTest/AnalyzerFileReferenceTests.cs

```diff
@@ -223,6 +223,8 @@ public class TestAnalyzer : DiagnosticAnalyzer
 
             dir.CopyFile(typeof(System.Reflection.Metadata.MetadataReader).Assembly.Location);
             dir.CopyFile(typeof(AppDomainUtils).Assembly.Location);
+            dir.CopyFile(typeof(Memory<byte>).Assembly.Location);
```

Comment:
>byte [](start = 39, length = 4)

nit: the `byte` is probably not needed: `typeof(Memory<>)` (here and in `TestHelpers.cs`)